react-ui/.neutrinorc.js

```diff
@@ -0,0 +1,41 @@
+const DEFAULT_HOST = 'localhost';
+const DEFAULT_PORT = 9000;
+const port = process.env.PORT || DEFAULT_PORT;
+
+module.exports = {
+  options: {
+    root: __dirname,
+  },
+  use: [
+    '@mozilla-frontend-infra/react-lint',
+    [
+      '@neutrinojs/react',
+      {
+        devServer: {
+          host: process.env.HOST || DEFAULT_HOST,
+          port,
+        },
+        html: {
+          title: 'Balrog Admin',
+          favicon: `${__dirname}/src/images/favicon.png`
+        },
+        env: {
+          HOST: DEFAULT_HOST,
+          PORT: DEFAULT_PORT,
+          BALROG_ROOT_URL: 'https://localhost:8010',
+          AUTH0_CLIENT_ID: 'GlZhJQfx52b7MLQ19AjuTJHieiB4oh1j',
+          AUTH0_DOMAIN: 'balrog-localdev.auth0.com',
+          AUTH0_AUDIENCE: 'balrog-localdev',
+          AUTH0_RESPONSE_TYPE: 'token id_token',
+          AUTH0_SCOPE: 'full-user-credentials openid profile email',
+          AUTH0_REDIRECT_URI: `http://localhost:${port}/login`,
```

Comment:
Maybe hardcode the url:port because it's whitelisted in auth0 and changing the port may cause confusions?

Reply:
Arguably, it's hardcoded already, just set at the top of the file?

Reply:
Yeah, it's hardcoded unless you run it with `PORT=xxx`. Not a big deal.